content/_guides/managing-flows.md

```diff
@@ -0,0 +1,147 @@
+
+Then click the corresponding buttons to start **(1)**, edit **(2)** or delete **(3)** Flow.
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_4.png)
+
+In case you need to stop a running Flow, click the stop button:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_5.png)
+
+
+
+#### Start, Stop, Edit, Delete via the API
+
+1\. To start a Flow via the API, we will use the following request:
+
+`POST {{apiBaseUri}}/v2/flows/{FLOW_ID}/start`
+
+Below are request parameters:
+
+| **URL Parameter**                    | **Required** | **Description**                                   |
+|----------------------------------|--------------|---------------------------------------------------|
+| `FLOW_ID`                             | yes          | Flow ID |
+
+2\. To stop a Flow via the API, we will use the following request:
+
+`POST {{apiBaseUri}}/v2/flows/{FLOW_ID}/stop`
+
+Below are request parameters:
+
+| **URL Parameter**                    | **Required** | **Description**                                   |
+|----------------------------------|--------------|---------------------------------------------------|
+| `FLOW_ID`                             | yes          | Flow ID |
+
+3\. To delete a Flow via the API, we will use the following request:
+
+`DELETE {{apiBaseUri}}/v2/flows/{FLOW_ID}`
+
+Below are request parameters:
+
+| **URL Parameter**                    | **Required** | **Description**                                   |
+|----------------------------------|--------------|---------------------------------------------------|
+| `FLOW_ID`                             | yes          | Flow ID |
+
+4\. To edit a Flow to some extent via the API, we will use the following request:
+
+`PATCH {{apiBaseUri}}/v2/flows/{FLOW_ID}`
+
+Below are request parameters:
+
+| **URL Parameter**                    | **Required** | **Description**                                   |
+|----------------------------------|--------------|---------------------------------------------------|
+| `FLOW_ID`                             | yes          | Flow ID |
+
+
+| **Payload Parameter**                    | **Required** | **Description**                                   |
+|----------------------------------|--------------|---------------------------------------------------|
+| `type`                             | yes          | Allowed value: `flow` |
+| `id`                             | yes          | Flow ID |
+| `attributes.name`                             | no         | Flow name |
+| `attributes.type`                             | no          | Flow type. Allowed values: `ordinary`, `long_running `|
+| `attributes.graph`                             | no          |  	Flow graph representing component connections |
+| `attributes.cron`                             | no          | CRON expression representing Flow timing |
+
+
+## Copy Flow, Switch Flow Type
```

Comment:
Maybe better to split these two topics into 2 sections: `Copying Flow` and `Switching Flow Type`?

BTW why `Copy Flow` and not `Copying Flow`. I see many docs using present continuous tense.

Reply:
Copy Flow is, basically, the name of the process of Copying a Flow. Thus, when we say "Copying", we mean the process. It's one of the things that have zero chance of confusing a reader.